stl/src/filesystem.cpp

```diff
@@ -793,43 +793,13 @@ _Success_(return == __std_win_error::_Success) __std_win_error
     return {_Size, __std_win_error::_Success};
 }
 
-// This structure is meant to be embedded into __std_fs_stats that properly aligned it,
-// so that 64-bit values are fully aligned. Note that _File_size fields are flipped to be in low:high order
-// and represented by __std_fs_filetime which is a pair of ulongs.
-// If this structure is used in GetFileAttributesEx, after successful read, File_size parts must be put in
-// low:high order, as GetFileAttributesEx returns them in high:low order.
-struct _File_attr_data { // typedef struct _WIN32_FILE_ATTRIBUTE_DATA {
-    __std_fs_file_attr _Attributes; //     DWORD dwFileAttributes;
-    __std_fs_filetime _Creation_time; //     FILETIME ftCreationTime;
-    __std_fs_filetime _Last_access_time; //     FILETIME ftLastAccessTime;
-    __std_fs_filetime _Last_write_time; //     FILETIME ftLastWriteTime;
-    unsigned long _File_size_high; //     DWORD nFileSizeHigh;
-    unsigned long _File_size_low; //     DWORD nFileSizeLow;
-}; // } WIN32_FILE_ATTRIBUTE_DATA, *LPWIN32_FILE_ATTRIBUTE_DATA;
-
-struct alignas(long long) _Aligned_file_attrs {
-    unsigned long _Padding; // align the __std_fs_filetime inside _Data to make the memcpy below an ordinary 64-bit load
-    _File_attr_data _Data;
-
-    [[nodiscard]] long long _Last_write_time() const noexcept {
-        long long _Result;
-        _CSTD memcpy(&_Result, &_Data._Last_write_time, sizeof(_Result));
-        return _Result;
-    }
-
-    [[nodiscard]] unsigned long long _File_size() const noexcept {
-        return (static_cast<unsigned long long>(_Data._File_size_high) << 32) + _Data._File_size_low;
-    }
-};
+static unsigned long long _Merge_to_ull(unsigned long _Low, unsigned long _High) noexcept {
+    return static_cast<unsigned long long>(_Low) | static_cast<unsigned long long>(_High) << 32;

```

Comment:
Because the bitwise operators have notoriously confusing precedence for humans, we conventionally add extra parentheses, even when technically not required by the syntax. Here, the `<<` should be wrapped in parens.